common/lib/xmodule/xmodule/partitions/partitions.py

```diff
@@ -104,6 +104,9 @@ class UserPartition(namedtuple("UserPartition", "id name description groups sche
     # The default scheme to be used when upgrading version 1 partitions.
     VERSION_1_SCHEME = "random"
 
+    RANDOM_SCHEME = "random"
```

Comment:
I'm not sure about this. The intention is that the schemes are added as plugins in other Django apps, so the core code is not meant to know about their existence. However, this decision has proved hard to work with, and there are places where we have hard coded the constants. @cahrens what do you think?